debian/control

```diff
@@ -0,0 +1,22 @@
+Source: libplayback
+Section: libs
+Priority: optional
+Maintainer: Ivaylo Dimitrov <ivo.g.dimitrov.75@gmail.com>
```

Comment:
If you are going to maintain this Debian packaging (we would be happy if yes :-)) then fill free to put your name there.